xarray/tests/test_dataarray.py

```diff
@@ -3374,6 +3374,18 @@ def test_to_and_from_series(self):
         expected_da = self.dv.rename(None)
         assert_identical(expected_da, DataArray.from_series(actual).drop(["x", "y"]))
 
+    def test_from_series_sparse(self):
+        sparse = pytest.importorskip("sparse")
```

Comment:
I find it would be more readable to write
```python
@requires_sparse
def test_from_series_sparse(self):
    import sparse
```

Reply:
Agreed (and changed here), though I'm not sure it's worth enforcing this in general since it goes slightly against the norms of pytest.